convert2rhel/pkgmanager/handlers/yum.py

```diff
@@ -220,6 +220,9 @@ def _process_transaction(self, validate_transaction):
             pkgmanager.Errors.YumRPMCheckError,
             pkgmanager.Errors.YumTestTransactionError,
             pkgmanager.Errors.YumRPMTransError,
+            pkgmanager.Errors.YumDownloadError,
+            pkgmanager.Errors.YumBaseError,
+            pkgmanager.Errors.YumGPGCheckError,
         ) as e:
```

Comment:
```suggestion
            pkgmanager.Errors.YumBaseError
        ) as e:
```

`YumBaseError` is the base class for all of the yum exceptions so we can just list that to catch all of these exceptions.  Note that this could be made into a single line but github's suggestion feature wouldn't let me select all the way to the `except (` line.  That would look like this:

``` python
        except pkgmanager.Errors.YumBaseError as e:
```